data/test/vtgate/aggr_cases.txt

```diff
@@ -272,6 +272,51 @@
   }
 }
 
+# group by a unique vindex should revert to simple route, even if aggr is complex
```

Comment:
For completeness, you should add a couple more unsupported test cases:

One to show that complex aggregates are unsupported when grouping by a field that is not in a vindex at all, and one when grouping by a field that is in a non-unique vindex.